torch/testing/_internal/common_methods_invocations.py

```diff
@@ -6080,6 +6080,11 @@ def gradcheck_wrapper_triangular_input(op, input, *args, upper=False, **kwargs):
                    dtypesIfCUDA=all_types_and(torch.bool, torch.half, torch.bfloat16),
                    sample_inputs_func=sample_inputs_logit,
                    safe_casts_outputs=True),
+    UnaryUfuncInfo('erfcx',
+                   ref=scipy.special.erfcx if TEST_SCIPY else _NOTHING,
+                   aliases=('erfcx', ),
```

Comment:
The "aliases" field is only for operations which have an alias like how torch.det actually calls torch.linalg.det, so we won't need it here